views/_includes/adobe.njk

```diff
@@ -6,8 +6,8 @@
 <meta name="dcterms.service" content="ESDC-EDSC_c19b-c19p"/>
 
 {% if $env.ADOBE_ENV === 'production' %}
-  <script src="https://assets.adobedtm.com/launch-EN0cf6c2810a2b48f8a4c36502a1b09541.min.js"></script>
+  <script src="//assets.adobedtm.com/launch-EN0cf6c2810a2b48f8a4c36502a1b09541.min.js"></script>
 {% else %}
-  <script src="https://assets.adobedtm.com/launch-EN11c0261481f74c56b7656937bbd995e9-staging.min.js"></script>
+  <script src="//assets.adobedtm.com/launch-EN11c0261481f74c56b7656937bbd995e9-staging.min.js"></script>
```

Comment:
I think `//` will default to current protocol, so if you're on `https://covid-benefits...` it'll call out to `https://assets.adobe....` is that what's intended?

Reply:
Yeah that's what is documented in the installation document I figured it was better to just follow the instructions then having to continuously explain to the analytics team that I understand how it works.